content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,121 @@
+There are still other minor regressions that need to be ironed out until 0.4.15 reaches the final stage for a release.
+
+Due to development resources and manpower constraints we cannot give an exact estimation when 0.4.15 will be released to the general public but we will eventually get there. We are sorry for the inconvenience!
+In the meantime you can try [nightly builds](https://reactos.org/getbuilds) that pack up with latest bugfixes, new features and whatnot but expect also new bugs and such!
+
+## Kernel Debugger overhaul
+Historically, ReactOS was built with the GCC compiler, and to help with kernel development an in-built local kernel debugger, KDBG, was added. This debugger possesses its own set of commands and syntax, can understand only an in-house debugging-symbols format (“rossym”) – it does not support PDB. However, being a local debugger, it is able to interact with the user via a serial port connection or locally via screen display; input can be done either via serial or via keyboard (PS/2).
+Later on, the possibility to compile ReactOS with MSVC was added, and with it too, support for the WinDbg debugger. Kernel debugging with WinDbg works by having a debugging stub (KD64, this is a similar idea to GDB) on the target, that is capable of manipulating the state of the OS, and having the actual debugger offloaded to a host computer.
+
+Hermès is working on improving the ReactOS' kernel debugger (KDBG) with new features and miscellaneous fixes so that it makes easier for developers and testers alike to reliably debug ReactOS.
+The following improvements are:
+* Isolating terminal and input-related functionalities from the rest of KDBG ([#5188](https://github.com/reactos/reactos/pull/5188));
+* Fixed an input buffer reading bug in `KdbpReadCommand()` and `KdReceivePacket()` routines ([#4914](https://github.com/reactos/reactos/pull/4914));
+* Reworked the boot-phase initialization of KD and KDBG ([#4892](https://github.com/reactos/reactos/pull/4892));
+* Moved the code handling of Dmesg buffer from screen provider to KDBG provider ([#5143](https://github.com/reactos/reactos/pull/5143));
+
+Eventually the goal of this work is to have a terminal version of KDCOM and separate KDBG into a dedicated KDCOM-like extension DLL, such like `kdrosdbg.dll` (see ([#4600](https://github.com/reactos/reactos/pull/4600))).
+All of this effort is further coordinated with Timo Kreuzer.
+
+## UEFI Boot
+Thanks to the work of Timo Kreuzer (tkreuzer) our AMD64 is now capable of booting on an increasing number of hardware.
+With this new step we now are able to look at booting on more modern devices, the first step of this transition being the support of UEFI Class 3 boot (Systems without a BIOS fallback).
+Work has been going on since the beginning of the year to transition freeloader our default bootloader for ReactOS to support UEFI on ARM32, ARM64, x86 and of course primarily; AMD64.
```

Comment:
```suggestion
Work has been underway since the beginning of the year to transition FreeLoader, our default bootloader for ReactOS, to support UEFI on ARM32, ARM64, x86 and of course: AMD64.
```